kernel/src/main/scala/org/apache/toree/kernel/api/Kernel.scala

```diff
@@ -403,15 +414,16 @@ class Kernel (
       case Some(master) if !master.contains("local") =>
         // when connecting to a remote cluster, the first call to getOrCreate
         // may create a session and take a long time, so this starts a future
-        // to get the session. if it take longer than 100 ms, then print a
+        // to get the session. if it take longer than specified timeout, then print a
```

Comment:
maybe you should still mention that (100 ms is default)

Reply:
Updated the comment.